docs/reference/commandline/ps.md

```diff
@@ -51,6 +51,7 @@ The currently supported filters are:
 * exited (int - the code of exited containers. Only useful with `--all`)
 * status (created|restarting|running|paused|exited)
 * ancestor (`<image-name>[:<tag>]`,  `<image id>` or `<image@digest>`) - filters containers that were created from the given image or a descendant.
+* isolation (default|hyperv)
```

Comment:
Urm - re https://github.com/docker/docker/pull/17534#discussion_r43998683, this is docs/reference/commandline/ps.md ;) But updated (pushing soon)